pynicotine/gtkgui/icons/hicolor/128x128/apps/org.nicotine_plus.Nicotine.png.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2025 Nicotine+ Contributors
```

Comment:
It should reflect the actual content which is "2020-2024" in this case, since this .license text file itself isn't what's being licensed here.